File: projects/order-creation-queries/core/src/main/java/com/robinfood/core/entities/OrderDetailProductGroupEntity.java
package com.robinfood.core.entities;

import lombok.Data;

import java.util.List;

@Data
public class OrderDetailProductGroupEntity {

    private final Long id;

    private final String name;

    private final List<OrderDetailProductGroupPortionEntity> portions;

    private final List<OrderDetailRemovedPortionEntity> removedPortions;

    private final String sku;
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/entities/OrderDetailProductGroupEntity.kt
package com.robinfood.core.entities

data class OrderDetailProductGroupEntity(
        val id: Long?,
        val name: String?,
        val portions: List<OrderDetailProductGroupPortionEntity>?,
        val removedPortions: List<OrderDetailGroupRemovedPortionEntity>?,
        val sku: String?
)
